src/js/main.js

```diff
@@ -1,27 +1,3 @@
 import { loadHeaderFooter } from "./utils.mjs";
-import ProductData from "./ProductData.mjs";
-import ProductListing from "./ProductList.mjs";
 
 loadHeaderFooter();
-
-const productData = new ProductData("tents");
-
-async function loadProducts() {
-  try {
-    const products = await productData.getProducts();
-    console.log(products); // Check if products are logged correctly
-
-    const listElement = document.querySelector(".product-list"); // Ensure this selector matches your HTML structure
-
-    const productListing = new ProductListing(
-      "Tents",
-      productData,
-      listElement,
-    );
-    await productListing.init();
-  } catch (error) {
-    console.error("Error loading products:", error);
-  }
-}
-
-loadProducts();
```

Comment:
This was all old code from week 1 that is no longer relevant. It just caused console errors.